### DISCUSSION TITLE
v0.11.0

### DISCUSSION THREAD

**Discussion by romange** [dragonflydb/dragonfly#487] [Announcements]
# Dragonfly v0.11.0 - Thanksgiving edition

Some prominent changes:
* You can not pass `--maxmemory=16G` flag instead of using bytes integer.
*  Monitor command is implemented, it's also much faster than in Redis. 
*  Missing options in existing commands were added. Amazing work by our contributors. 
*  Significant progress was made with the replication. Still, lots needs to be done in this area. Expected release for this feature - Q1 23.

# Huge thanks to all the contributors! ❤️ 
## What's Changed
* feat(server): sending and dispatching of messages using different internals by @boazsade in https://github.com/dragonflydb/dragonfly/pull/424
* ci(workflow): fixing discord notification link by @ranrib in https://github.com/dragonflydb/dragonfly/pull/419
* feat(server): Allow sampling arena stats from a specified thread by @romange in https://github.com/dragonflydb/dragonfly/pull/428
* Monitor command by @boazsade in https://github.com/dragonflydb/dragonfly/pull/427
* feat(server): Multi rdb load by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/430
* refactor(server): Refactor and tidy dflycmd by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/432
* feat(server): monitor command should return OK on creation (#344) by @boazsade in https://github.com/dragonflydb/dragonfly/pull/433
* refactor(server): Separate start & stop in rdb save by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/437
* fix: dragonfly compilation failure due to glibc version less than 2.30 by @Super-long in https://github.com/dragonflydb/dragonfly/pull/434
* feat(server): monitor command - cover by tests by @boazsade in https://github.com/dragonflydb/dragonfly/pull/435
* fix: "wrong number of arguments" must be in lower case by @lsvmello in https://github.com/dragonflydb/dragonfly/pull/439
* refactor(tests): Refactor pytest by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/449
* docs(build): Canonicalize build from source instructions. Add CLA terms by @romange in https://github.com/dragonflydb/dragonfly/pull/454
* feat(server): implement TOUCH command by @lsvmello in https://github.com/dragonflydb/dragonfly/pull/444
* feat(server): implement json.mget command (#104) by @iko1 in https://github.com/dragonflydb/dragonfly/pull/455
* feat(server): Support for LIMIT and REV in ZRANGE #422 by @RedhaL in https://github.com/dragonflydb/dragonfly/pull/456
* fix(list): Fixes blpop failure. by @romange in https://github.com/dragonflydb/dragonfly/pull/462
* ci: added clang pre-commit hook by @lsvmello in https://github.com/dragonflydb/dragonfly/pull/461
* feat(tests): ioredis being able to successfully run tests, make it ready to be part of CI by @boazsade in https://github.com/dragonflydb/dragonfly/pull/459
* feat(server): Basic capped full sync by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/440
* feat(server) implement pexpire command by @b0bleet in https://github.com/dragonflydb/dragonfly/pull/464
* server: implemented json.arrappend command by @b0bleet in https://github.com/dragonflydb/dragonfly/pull/460
* feat(server): simplify handling of the pub message handling by @boazsade in https://github.com/dragonflydb/dragonfly/pull/465
* refactor(server): Refactor replica by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/472
* fix(server): reject eval inside transaction multi blocks #457 by @boazsade in https://github.com/dragonflydb/dragonfly/pull/471
* feat(server): implement json.debug command (#104) by @iko1 in https://github.com/dragonflydb/dragonfly/pull/463
* fix(server): issue with invalid command inside multi #468 by @boazsade in https://github.com/dragonflydb/dragonfly/pull/469
* SSCAN command support count and match parameters by @adiholden in https://github.com/dragonflydb/dragonfly/pull/466
* bug(dense set): fix scan function to return only home bucket data by @adiholden in https://github.com/dragonflydb/dragonfly/pull/474
* fix: maxmemory flag support human-readable format by @lsvmello in https://github.com/dragonflydb/dragonfly/pull/476
* fix(server): Fix bad maxmemory casting by @romange in https://github.com/dragonflydb/dragonfly/pull/477
* fix(server): Fix snapshot saving bug by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/479
* feat(server): add support for KEEPTTL options with SET command #389 by @Super-long in https://github.com/dragonflydb/dragonfly/pull/478
* fix: DCHECK_NOTNULL gets WARNING when return value is not used. by @Super-long in https://github.com/dragonflydb/dragonfly/pull/483
* fix(server): Fix path concatenation in save by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/481
* feat(server): implement json.resp command (#104) by @iko1 in https://github.com/dragonflydb/dragonfly/pull/482
* feat(server family): add connection set name command fixes #458 by @adiholden in https://github.com/dragonflydb/dragonfly/pull/485

## New Contributors
* @Super-long made their first contribution in https://github.com/dragonflydb/dragonfly/pull/434
* @lsvmello made their first contribution in https://github.com/dragonflydb/dragonfly/pull/439
* @b0bleet made their first contribution in https://github.com/dragonflydb/dragonfly/pull/464
* @adiholden made their first contribution in https://github.com/dragonflydb/dragonfly/pull/466

**Full Changelog**: https://github.com/dragonflydb/dragonfly/compare/v0.10.0...v0.11.0

<hr /><em>This discussion was created from the release <a href='https://github.com/dragonflydb/dragonfly/releases/tag/v0.11.0'>v0.11.0</a>.</em>

**Comment by ChrisnaMishra** [comment]
Nice job

**Comment by ChrisnaMishra** [comment]
> # Dragonfly v0.11.0 - Thanksgiving edition
> 
> Some prominent changes:
> * You can not pass `--maxmemory=16G` flag instead of using bytes integer.
> *  Monitor command is implemented, it's also much faster than in Redis. 
> *  Missing options in existing commands were added. Amazing work by our contributors. 
> *  Significant progress was made with the replication. Still, lots needs to be done in this area. Expected release for this feature - Q1 23.
> 
> # Huge thanks to all the contributors! ❤️ 
> ## What's Changed
> * feat(server): sending and dispatching of messages using different internals by @boazsade in https://github.com/dragonflydb/dragonfly/pull/424
> * ci(workflow): fixing discord notification link by @ranrib in https://github.com/dragonflydb/dragonfly/pull/419
> * feat(server): Allow sampling arena stats from a specified thread by @romange in https://github.com/dragonflydb/dragonfly/pull/428
> * Monitor command by @boazsade in https://github.com/dragonflydb/dragonfly/pull/427
> * feat(server): Multi rdb load by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/430
> * refactor(server): Refactor and tidy dflycmd by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/432
> * feat(server): monitor command should return OK on creation (#344) by @boazsade in https://github.com/dragonflydb/dragonfly/pull/433
> * refactor(server): Separate start & stop in rdb save by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/437
> * fix: dragonfly compilation failure due to glibc version less than 2.30 by @Super-long in https://github.com/dragonflydb/dragonfly/pull/434
> * feat(server): monitor command - cover by tests by @boazsade in https://github.com/dragonflydb/dragonfly/pull/435
> * fix: "wrong number of arguments" must be in lower case by @lsvmello in https://github.com/dragonflydb/dragonfly/pull/439
> * refactor(tests): Refactor pytest by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/449
> * docs(build): Canonicalize build from source instructions. Add CLA terms by @romange in https://github.com/dragonflydb/dragonfly/pull/454
> * feat(server): implement TOUCH command by @lsvmello in https://github.com/dragonflydb/dragonfly/pull/444
> * feat(server): implement json.mget command (#104) by @iko1 in https://github.com/dragonflydb/dragonfly/pull/455
> * feat(server): Support for LIMIT and REV in ZRANGE #422 by @RedhaL in https://github.com/dragonflydb/dragonfly/pull/456
> * fix(list): Fixes blpop failure. by @romange in https://github.com/dragonflydb/dragonfly/pull/462
> * ci: added clang pre-commit hook by @lsvmello in https://github.com/dragonflydb/dragonfly/pull/461
> * feat(tests): ioredis being able to successfully run tests, make it ready to be part of CI by @boazsade in https://github.com/dragonflydb/dragonfly/pull/459
> * feat(server): Basic capped full sync by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/440
> * feat(server) implement pexpire command by @b0bleet in https://github.com/dragonflydb/dragonfly/pull/464
> * server: implemented json.arrappend command by @b0bleet in https://github.com/dragonflydb/dragonfly/pull/460
> * feat(server): simplify handling of the pub message handling by @boazsade in https://github.com/dragonflydb/dragonfly/pull/465
> * refactor(server): Refactor replica by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/472
> * fix(server): reject eval inside transaction multi blocks #457 by @boazsade in https://github.com/dragonflydb/dragonfly/pull/471
> * feat(server): implement json.debug command (#104) by @iko1 in https://github.com/dragonflydb/dragonfly/pull/463
> * fix(server): issue with invalid command inside multi #468 by @boazsade in https://github.com/dragonflydb/dragonfly/pull/469
> * SSCAN command support count and match parameters by @adiholden in https://github.com/dragonflydb/dragonfly/pull/466
> * bug(dense set): fix scan function to return only home bucket data by @adiholden in https://github.com/dragonflydb/dragonfly/pull/474
> * fix: maxmemory flag support human-readable format by @lsvmello in https://github.com/dragonflydb/dragonfly/pull/476
> * fix(server): Fix bad maxmemory casting by @romange in https://github.com/dragonflydb/dragonfly/pull/477
> * fix(server): Fix snapshot saving bug by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/479
> * feat(server): add support for KEEPTTL options with SET command #389 by @Super-long in https://github.com/dragonflydb/dragonfly/pull/478
> * fix: DCHECK_NOTNULL gets WARNING when return value is not used. by @Super-long in https://github.com/dragonflydb/dragonfly/pull/483
> * fix(server): Fix path concatenation in save by @dranikpg in https://github.com/dragonflydb/dragonfly/pull/481
> * feat(server): implement json.resp command (#104) by @iko1 in https://github.com/dragonflydb/dragonfly/pull/482
> * feat(server family): add connection set name command fixes #458 by @adiholden in https://github.com/dragonflydb/dragonfly/pull/485
> 
> ## New Contributors
> * @Super-long made their first contribution in https://github.com/dragonflydb/dragonfly/pull/434
> * @lsvmello made their first contribution in https://github.com/dragonflydb/dragonfly/pull/439
> * @b0bleet made their first contribution in https://github.com/dragonflydb/dragonfly/pull/464
> * @adiholden made their first contribution in https://github.com/dragonflydb/dragonfly/pull/466
> 
> **Full Changelog**: https://github.com/dragonflydb/dragonfly/compare/v0.10.0...v0.11.0
> 
> <hr /><em>This discussion was created from the release <a href='https://github.com/dragonflydb/dragonfly/releases/tag/v0.11.0'>v0.11.0</a>.</em>

Nice

**Reply by romange** [reply]
Thanks!

**Reply by EricHill10** [reply]
I don't want
